### PR TITLE
Instructions on how to add package controller

### PR DESCRIPTION
**Pull request by TerryHowe**
Closes: https://github.com/aws/eks-anywhere/issues/4832

### CODE REVIEW

File: docs/content/en/docs/tasks/packages/_index.md [hunk 1]
```diff
@@ -132,3 +132,58 @@ eksctl anywhere generate package harbor --cluster ${CLUSTER_NAME} --kube-version
...
+  name: billy
+spec:
+  packages:
+    disable: false
+```
+
+You may need to create or update your credentials which you can do with a command like this. Set the environment variables to the proper values before running the command.
+```bash
+kubectl delete secret -n eksa-packages aws-secret
+kubectl create secret -n eksa-packages generic aws-secret --from-literal=AWS_ACCESS_KEY_ID=${EKSA_AWS_ACCESS_KEY_ID} --from-literal=AWS_SECRET_ACCESS_KEY=${EKSA_AWS_SECRET_ACCESS_KEY}  --from-literal=REGION=${EKSA_AWS_REGION}
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/packages/_index.md:182]: Is this whole command on a single line? If possible, could you break up the line with backslashes?

**Review by chrisnegus — COMMENTED** [review]
Looks good. I had a question about one of the command lines.

File: docs/content/en/docs/tasks/packages/_index.md (hunk 1 above)
Comment (TerryHowe) [reply docs/content/en/docs/tasks/packages/_index.md:182]: That is two commands delete and create

Comment (chrisnegus) [reply docs/content/en/docs/tasks/packages/_index.md:182]: Oh. Sure. I meant the create command. I think someone could see the whole command better if you use backslashes (\) to break it up into three or four lines. It still works the way you have it, but I think some of it you would have to scroll to the right to see.

Comment (TerryHowe) [reply docs/content/en/docs/tasks/packages/_index.md:182]:
```suggestion
kubectl create secret -n eksa-packages generic aws-secret \
   --from-literal=AWS_ACCESS_KEY_ID=${EKSA_AWS_ACCESS_KEY_ID} \
   --from-literal=AWS_SECRET_ACCESS_KEY=${EKSA_AWS_SECRET_ACCESS_KEY}  \
   --from-literal=REGION=${EKSA_AWS_REGION}
```

Comment (TerryHowe) [reply docs/content/en/docs/tasks/packages/_index.md]: gotcha, fixed.

**Pull request merged**